src/main/java/seedu/address/model/person/Name.java

```diff
@@ -46,7 +46,7 @@ public String toString() {
 
     @Override
     public boolean equals(Object other) {
-        return other == this // short circuit if same object
```

Comment:
EH Y U REMOVE USEFUL COMMENT :(

Reply:
not sure why i deleted it, I think it was accidental.
Anyway fixed this in the new commit!